x/emissions/module/rewards/topic_skimming.go

```diff
@@ -48,7 +48,10 @@ func SkimTopTopicsByWeightDesc(weights map[TopicId]*alloraMath.Dec, N uint64, bl
 	for topicId := range weights {
 		topicIds = append(topicIds, topicId)
 	}
-
```

Comment:
we might not actually need this because the operations of `SortTopicsByWeightDescWithRandomTiebreaker()` don't involve a map-like data structure => could save compute to remove